README.md

```diff
@@ -163,6 +164,33 @@ let feedFromTwitter user pwd = taskSeq {
+|----------------------------|---------------------------------------------------------------------------------------|----------------------------------------------------------------------|
+| **Frameworks**             | .NET 5.0+, NetStandard 2.1                                                        | .NET 5.0+, NetStandard 2.0 and 2.1, .NET Framework 4.6.1+            |
+| **Underlying type**        | `System.Collections.Generic.IAsyncEnumerable<'T>`                                 | Its own type, also called `IAsyncEnumerable<'T>`, but not compatible |
+| **Implementation**         | State machine (statically compiled)                                               | No state machine, continuation style                                 |
+| **Semantics**              | `seq`-like: on-demand                                                             | `seq`-like: on-demand                                                |
+| **Support `let!`**         | All `task`-like: `Async<'T>`, `ValueTask<'T>`, `Task<'T>` or any `GetAwaiter()`   | `Async<'T>` only                                                     |
+| **Support `do!`**          | `Async<unit>`, `Task<unit>` and `Task`, `ValueTask<unit>` and `ValueTask`         | `Async<unit>` only                                                   |
+| **Support `yield!`**       | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence                                  | `AsyncSeq`                                                           |
+| **Support `for`**          | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence                                  | `AsyncSeq` any sequence                                              |
+| **Behavior with `yield`**  | Zero allocations, no `Task` or even `ValueTask` created                           | Allocated, an F# `Async` wrapped in a singleton `AsyncSeq`           |
+| **Conversion to other**    | `TaskSeq.toAsyncSeq`                                                              | `AsyncSeq.toAsyncEnum`                                               |
+| **Conversion from other**  | Implicit (yield!) or `TaskSeq.ofAsyncSeq`                                         | `AsyncSeq.ofAsyncEnum`                                               |
+| **Recursion in `yield!`**  | No (requires F# support, upcoming)                                                | Yes                                                                  |
+| **Based on F# concept of** | `task`                                                                            | `async`                                                              |
+| **`MoveNextAsync`**        | `ValueTask<bool>`                                                                 | `Async<'T option>`                                                   |
+| **`Current` internals**    | `ValueOption<'T>`                                                                 | `Option<'T>`                                                         |
```

Comment:
I would group these 2 in a single MoveNext/Current row. I think it's clearer as in AsyncSeq both things are in a single type.

Reply:
Tx. I'll probably just drop the second line altogether. Primarily, I don't want the left-most column to be too wordy and I can't think of a good way to explain both. Curious people can always look it up.

Reply:
Done, tx.